tee-worker/ts-tests/README.md

```diff
@@ -42,7 +42,9 @@ Bulk II identity test: `corepack yarn test-bulk-identity:local`
 
 Bulk II vc test: `corepack yarn test-bulk-vc:local`
 
-DI identity test: `corepack yarn test-identity-direct-invocation:local`
+Direct invocation substrate identity test: `corepack yarn test-substrate-di-identity:local`
```

Comment:
Please keep it consistent, there is DI left in line 49, also II is left unchanged.
Personally i prefer DI and II